packages/subgraph/schema.graphql

```diff
@@ -196,13 +196,24 @@ enum VoteOption {
   No
 }
 
+enum VoteMode {
+  Standard
+  EarlyExecution
+  VoteReplacement
+}
+
+type MajorityVotingSettings @entity {
```

Comment:
is that empty entity necessary? I don't see it being used anywhere

Reply:
No. This was a left accidentally. Removed in 5a1b1b2be532fdd4d68d9d47ce6b66bea781457f